client/src/components/NavigationBar.tsx

```diff
@@ -0,0 +1,48 @@
+import { Container, Nav, Navbar } from "react-bootstrap";
+import { useLocation, useNavigate } from "react-router-dom";
+import { PersonCircle } from "react-bootstrap-icons";
+import { NAV_CONFIG } from "../App.tsx";
+
+function NavigationBar() {
+  const location = useLocation().pathname;
+  const navigate = useNavigate();
+  return (
+    <Navbar expand="lg" className="bg-body-tertiary">
+      <Container>
+        <Navbar.Brand href={"/"}>{NAV_CONFIG.brand}</Navbar.Brand>
+        <Navbar.Toggle aria-controls="basic-navbar-nav" />
+        <Navbar.Collapse id="basic-navbar-nav">
+          <Nav className="me-auto">
+            {NAV_CONFIG.routes.map((r) => (
```

Comment:
changing r to route would improve readability

Reply:
Fixed